ddtrace/appsec/_iast/_patch_modules.py

```diff
@@ -7,6 +7,7 @@
     "header_injection": True,
     "weak_cipher": True,
     "weak_hash": True,
+    "stacktrace_leak": True,
```

Comment:
We don't need this line, we use this dictionary when we want to patch with `ddtrace/appsec/_iast/taint_sinks/[element-in-dictionary].py::patch()`